webapi/src/test/scala/org/knora/webapi/e2e/v1/ResourcesV1R2RSpec.scala

```diff
@@ -1249,5 +1257,46 @@ class ResourcesV1R2RSpec extends R2RSpec {
 
 
         }
+        "create a resources from simple xml" in {
+
+            val params =
+                s"""<xml xmlns:beol="http://www.knora.org/ontology/beol"
+                   |   xmlns:biblio="http://www.knora.org/ontology/biblio">
+                   |   <beol:person id="abel">
+                   | 	    <beol:hasGivenName>Niels Henrik</beol:hasGivenName>
+                   |	      <beol:hasFamilyName>Abel</beol:hasFamilyName>
+                   |	  </beol:person>
+                   |   <biblio:Journal id="math_intelligencer_">
+                   |		    <biblio:hasName>math intelligencer </biblio:hasName>
+                   |   </biblio:Journal>
+                   |   <biblio:JournalArticle id="strings_in_the_16th_and_17th_centuries" >
+                   |       <biblio:publicationHasTitle>Strings in the 16th and 17th Centuries</biblio:publicationHasTitle>
+                   |       <biblio:publicationHasAuthor>
+                   |           <beol:person ref="abel"/>
+                   |       </biblio:publicationHasAuthor>
+                   |       <biblio:isPartOfJournal>
+                   |           <biblio:Journal ref="math_intelligencer_"/>
+                   |       </biblio:isPartOfJournal>
+                   |       <biblio:journalVolume>27</biblio:journalVolume>
+                   |       <biblio:startPage>48</biblio:startPage>
+                   |       <biblio:endPage>73</biblio:endPage>
+                   |       <biblio:publicationHasDate>1974</biblio:publicationHasDate>
+                   |    </biblio:JournalArticle>
+
+                   |</xml>""".stripMargin
+            Post("/v1/resources/xml", HttpEntity(ContentTypes.`text/xml(UTF-8)`, params)) ~> addCredentials(BasicHttpCredentials(biblioUserEmail, password)) ~> resourcesPath ~> check {
+                assert(status == StatusCodes.OK, response.toString)
+
+
+                val resourcesToCreate =Seq.empty[OneOfMultipleResourceCreateRequestV1]
+                val responseExpected = MultipleResourceCreateRequestV1( resourcesToCreate,
```

Comment:
Please have the test load the resources that were created (by making a `Get` request) and check the contents of each one.